commands/other/311/311_schedule.subcmd.js

```diff
@@ -62,7 +62,7 @@ module.exports = async function (client, interaction) {
             .setColor('YELLOW')
             .setTitle(`[${jour}] ${day}`)
             .setThumbnail(`${interaction.member.user.displayAvatarURL({dynamic: true, size: 32})}`)
-            .setDescription("No school today!");
+            .setDescription(`:hourglass: There are ${days_to_frc} days remaining before the beginning of the FRC!\n\nNo school today!`);
```

Comment:
change to `:calendar_spiral: No school today!` for consistency with the rest of the code

Reply:
I'll do it myself